src/_guides/language/effective-dart/design.md

```diff
@@ -952,13 +975,76 @@ class CallChainVisitor {
 }
 {% endprettify %}
 
+This guideline is deliberately softer than the previous one because the benefits
+of static typing are slightly reduced when code is confined within a single
+library. In practice, though, most Dart users type annotate all declarations,
+public or private.
+
+
+### AVOID annotating types for initialized local variables.
```

Comment:
Could we emphasize "initialized" (say, styling it with italics)?

Reply:
I tried to figure out a way to do the emphasis using just word order, but I couldn't make it work.

Reply:
Ok